Cargo.toml

```diff
@@ -49,6 +49,7 @@ file-rotate = "0.7.4"
 tokio = { version = "1", features = ["rt"] }
 async-trait = "0.1.68"
 reqwest = { version = "0.11", features = ["json"] }
+lapin = "2.2.1"
```

Comment:
Have we been moving away from optional dependencies turned on with feature flags? I guess I may be out of the loop with some v2 stuff.

Reply:
thanks for letting me know. I will adjust and also change the webhook sink to use the flag in another PR